src/MessageHandler.py

```diff
@@ -0,0 +1,112 @@
+import json
+import urllib2
+from src.MessageRouter import MessageRouter
+
+
+def new_message_handler(config):
+    def _new_command_handlers():
+        return {
+            'casts': _new_casts_handler(),
+            'streams': new_get_streams_response(fetch_url_contents),
+            'nyan': lambda message: ['~=[,,_,,]:3']
```

Comment:
You can say that, though I cannot do something with it unless you make clear how this would still be dynamic. I'm not going to refactor something in a state that seems static to me, just because you say it is actually dynamic without me understanding why.